### ISSUE TITLE
Chromium does not differentiate between website and extension permissions when it comes to clipboard access

### ISSUE THREAD

**Issue by 29edde2e** [daijro/Clipboard2File-Chrome#1]
When I tell people that Clipboard2File can't be ported to Chromium, I mean it can't be ported *securely*. Let me explain:

When the user clicks a file input, Chromium will prompt the user to give the *website* permission to read their clipboard, not just the extension. So as months pass, users will inadvertently amass a gigantic list of websites which they have given permission to read their clipboard, which was not the user's intention, of course. 

I tried to work around this by reading the clipboard in the background, but navigator.clipboard does not work in the background page. Ignoring this security flaw, it's simply a bad user experience to get prompted on every new website.

So in summary: Never publish this fork to the Chrome Web Store, you will be putting uninformed users at risk. You should also probably add a disclaimer to the README that Chromium does not differentiate between website and extension permissions when it comes to Clipboard access in content scripts.

**Comment by daijro** [comment]
Hello, thank you for letting me know. This Chromium flaw had come to my attention earlier while I was testing.

I am currently looking into a "helper server" to retrieve the clipboard external to the browser. While this isn't ideal, it should hopefully get around the need to request clipboard permissions for every new website.

I will add a disclaimer that this project is non-production ready. Thanks!

**Comment by daijro** [comment]
This issue should now be fixed in [v4.0](https://github.com/daijro/Clipboard2File-Chrome/commit/3007c95c9191343e15e40959fd42785f707b7f6e).
